### PR TITLE
New Backpack Type: Messenger Bags!

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
image
From downstream with love! These are mechanically identical to a normal backpack or satchel, just with more drip.
Also a little bit of :o)
image
These will exist both in prefs and in drobes/lockers.
image
Sprites by Zydras.

## Why It's Good For The Game
We love fashion here. We love it so much. And most backpacks are too in-your-face to fit well with a lot of 'fits.

## Changelog
:cl: Nerev4r & Zydras
image: Adds messenger bags to character setup and drobes/lockers!
/:cl:

### CODE REVIEW

File: code/game/objects/items/storage/backpack.dm [hunk 1]
```diff
@@ -809,3 +809,90 @@
...
+
+/obj/item/storage/backpack/messenger/explorer
+	name = "explorer messenger bag"
+	desc = "A robust messenger bag for stashing your loot, as well as making a remarkably cute accessory for your drakebone armor."
+	icon_state = "messenger_explorer"
+	inhand_icon_state = "messenger_explorer"
+
+/obj/item/storage/backpack/messenger/cap
+	name = "captain's messenger bag"
+	desc = "An exclusive messenger bag for NanoTrasen officers, made of real whaleleather."
```
Comment (san7890) [inline code/game/objects/items/storage/backpack.dm:890]: image

**Review by san7890 — CHANGES_REQUESTED** [review]
linters are failing

File: code/game/objects/items/storage/backpack.dm (hunk 1 above)
Comment (GoldenAlpharex) [reply code/game/objects/items/storage/backpack.dm:890]: Could at least give them the suggested changes 😔
```suggestion
	desc = "An exclusive messenger bag for Nanotrasen officers, made of real whale leather."
```

Comment (Nerev4r) [reply code/game/objects/items/storage/backpack.dm]: thx both of u

**Comment by nikothedude** [comment]
typical nerevar W

**Comment by mc-oofert** [comment]
holy shit fallout new vegas

File: code/_globalvars/lists/flavor_misc.dm [hunk 2]
```diff
@@ -142,17 +142,21 @@ GLOBAL_LIST_INIT(security_depts_prefs, sort_list(list(
 #define DBACKPACK "Department Backpack"
 #define DDUFFELBAG "Department Duffel Bag"
 #define DSATCHEL "Department Satchel"
+#define DMESSENGER "Department Messenger Bag"
 #define GBACKPACK "Grey Backpack"
 #define GDUFFELBAG "Grey Duffel Bag"
 #define GSATCHEL "Grey Satchel"
+#define GMESSENGER "Grey Messenger Bag"
 #define LSATCHEL "Leather Satchel"
 GLOBAL_LIST_INIT(backpacklist, list(
```
Comment (Mothblocks) [inline code/_globalvars/lists/flavor_misc.dm:151]: Remove this list and the `random_backpack()` proc. If everything still compiles then commit it

**Comment by SimplySmiley123** [comment]
Honestly very cool.

**Comment by LemonInTheDark** [comment]
Should be smaller with speed boost (instant powergamer pick don'tdo this)
I am a bit upset that they're just a resprite. V boring needs more imagination.

**Comment by SomeoneYouProbablyKnow** [comment]
Yes but they LOOK cool. Anything that reduces the amount of backpack i have to see.

**Comment by Stalkeros2** [comment]
Not everything should have gameplay mechanics.

File: code/_globalvars/lists/flavor_misc.dm (hunk 2 above)
Comment (Nerev4r) [reply code/_globalvars/lists/flavor_misc.dm:151]: used elsewhere oomfie :(

**Review by Wallemations — APPROVED** [review]

**Comment by LemonInTheDark** [comment]
> Not everything should have gameplay mechanics.

Yes, but I think this should

**Comment by Nerev4r** [comment]
> Should be smaller with speed boost (instant powergamer pick don'tdo this) I am a bit upset that they're just a resprite. V boring needs more imagination. You can do better then this.

you have no idea how tempted i was to actually do that. like, half capacity and a lil speedboost.

**Comment by CliffracerX** [comment]
> Should be smaller with speed boost (instant powergamer pick don'tdo this) I am a bit upset that they're just a resprite. V boring needs more imagination. You can do better then this.

As cool as it would be I feel like satchels would be better-suited to that role given their size on the player sprite (i.e, tiny but out of the way), *and* giving any storage item a speed buff makes it, like- instant wins-by-default because getting to places faster is almost always more important than being able to haul a few extra boxes of goodies around.

**Comment by YehnBeep** [comment]
Honestly, if someone made a certain type of bag give a speedboost, I'd keep one on hand just for that. Even with a capacity of 0, I'd use it sometimes and just get by with pockets/belt/maybe dragging. Boosting speed seems like it would be very strong.

**Comment by nikothedude** [comment]
Just saying there's a precedent for bags being cosmetic in the satchels

**Comment by san7890** [comment]
> Yes, but I think this should. Would be better at the least

we can balance it in the future i just think they're neat for the time being though

**Comment by IgiariValkyr** [comment]
Would have preferred the sprites to be similar to the ones on Virgo, like the backpack I used to use in college.
dreammaker_2023-08-30_03-43-45

**Pull request merged**